e2e/stress/MemoryLeakTest/Program.cs

```diff
@@ -127,7 +127,7 @@ public async Task<int> Method_LongRunningLeakTest_WrongEndpoint_Ok(TransportType
                 catch (TimeoutException) { }
                 catch (SocketException) { }
                 catch (TaskCanceledException) { }
-                catch (IotHubCommunicationException) { }
+                catch (IotHubCommunicationException ex2) { Console.WriteLine(ex2); }
                 finally
```

Comment:
Is it a good idea to print it to console

Reply:
This is 100% based on the other test already in the stress solution. I rather follow the same pattern so we don't loose focus on the purpose of testing this change.